From 7153578a234068c510e1a95453b1498a1d087a5c Mon Sep 17 00:00:00 2001
From: Valentin Churavy <v.churavy@gmail.com>
Date: Wed, 29 Sep 2021 18:04:10 -0400
Subject: [PATCH] Revert "[TSan] Add CMake check for minimal SDK requirements
 on Darwin"

This reverts commit 3bf3996cd4ef2d4898b32d4cef52d6549bbe6820.
---
 compiler-rt/lib/tsan/CMakeLists.txt | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/compiler-rt/lib/tsan/CMakeLists.txt b/compiler-rt/lib/tsan/CMakeLists.txt
index a60c8f842a2d..82117c62cf10 100644
--- a/compiler-rt/lib/tsan/CMakeLists.txt
+++ b/compiler-rt/lib/tsan/CMakeLists.txt
@@ -114,14 +114,6 @@ if("${CMAKE_C_FLAGS}" MATCHES "-Wno-(error=)?unused-command-line-argument")
 endif()
 
 if(APPLE)
-  # Ideally we would check the SDK version for the actual platform we are
-  # building for here.  To make our lifes easier we assume the host SDK setup is
-  # sane and use the macOS SDK version as a proxy for aligned SDKs.
-  find_darwin_sdk_version(macosx_sdk_version "macosx")
-  if ("${macosx_sdk_version}" VERSION_LESS 10.12)
-    message(FATAL_ERROR "Building the TSan runtime requires at least macOS SDK 10.12 (or aligned SDK on other platforms)")
-  endif()
-
   add_asm_sources(TSAN_ASM_SOURCES
     rtl/tsan_rtl_amd64.S
     rtl/tsan_rtl_aarch64.S
-- 
2.33.0

